kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/create-domain-script.sh

```diff
@@ -10,4 +10,8 @@ export DOMAIN_HOME=${DOMAIN_HOME_DIR}
 # Create the domain
 wlst.sh -skipWLSModuleScanning ${CREATE_DOMAIN_SCRIPT_DIR}/create-domain.py
 
+echo "###### ISTIO_ENABLED=${ISTIO_ENABLED}"
 
+if [ "${ISTIO_ENABLED}" == "true" ]; then
+  wlst.sh -skipWLSModuleScanning ${CREATE_DOMAIN_SCRIPT_DIR}/istio-fix-domain.py
```

Comment:
should we combine istio-fix-domain.py logic into create-domain.py and save a wlst.sh call?

Reply:
As sample codes, the clear structure is prior to the runtime efficiency. And this wlst only run once when creating the domain, so one more wlst call should not affect performance a lot.